core/pom.xml

```diff
@@ -26,4 +26,40 @@
 
     <artifactId>core</artifactId>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>compile-jdk8</id>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                        <configuration>
+                            <release>8</release>
+                            <outputDirectory>${project.build.outputDirectory}_jdk8</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>jar-jdk8</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>jdk8</classifier>
+                            <classesDirectory>${project.build.outputDirectory}_jdk8</classesDirectory>
```

Comment:
May be a silly question but will this mean it gets released as core-2.1.0.jar and core-2.1.0_jdk8.jar? If so is the underscore the normal way of appending it?

Reply:
This is just the name of the directory to look for classes in (see line 42 where the build plugin is set to use this to put the classes in).
Those classes are packaged into a jar where the usual name is appended with a '-' and the value of `<classifier>`. So the end result is core-2.1.0-jdk8.jar

Reply:
Cool, good to know!